examples/stocks/README.md

```diff
@@ -7,4 +7,5 @@ Open them in Google Colab to jumpstart your development journey!
 
 | Notebooks                                       |                                                                                    Open in Google Colab                                                                                    |
 |:------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+| [Algorithmic Trading Bot with Chat GPT](trading_bot_chatgpt.ipynb)            | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/alpacahq/alpaca-py/blob/master/examples/stocks/trading_bot_chatgpt.ipynb)  |
```

Comment:
how about making a new section for this one similar to structure of other assets?